java-spring-boot/spring-data-jpa/checklist.md

```diff
@@ -0,0 +1,33 @@
+# Spring Data JPA
+
+## Lombok
+
+Lombok is a great tool that makes our code cleaner and easier to read. 
+
+Use Lombok annotation to generate constructors, getters, setters and other things. 
+
+Take a look at these annotations (great source - https://projectlombok.org/features/): 
+1. `@Data`
+2. `@Getter`
+3. `@Setter`
+4. `@ToString`
+5. `@EqualsAndHashcode`
+6. `@NoArgsConstructor`
+7. `@RequiredArgsConstructor`
+9. `@AllArgsConstructor`
+
+**BUT**, it should be used with caution, because it can cause performance issues. 
+Consider these articles: 
+1. https://habr.com/ru/company/haulmont/blog/564682/
+2. https://thorben-janssen.com/lombok-hibernate-how-to-avoid-common-pitfalls
```

Comment:
I wouldn't add it to the checklist, because using Lombok is more like advice, it's not a mistake

Reply:
Would it be better if I rewrite this paragraph, so it's more clear that Lombok is optional and this is some tips if you decide to use it?